sandy/sampling.py

```diff
@@ -599,6 +599,11 @@ def parse(iargs=None):
                         metavar="{1,..,999}",
                         help="draw samples only from the selected MT sections "
                              "(default = keep all)")
+    parser.add_argument('--pdf',
+                        type=str,
+                        default='normal',
+                        help="draw samples according to the chosen distribution "
```

Comment:
Mention what options you have.
In theory you should also check that only one of the selected options is chosen. And ot should be co verteld to lowercase.

The parser allows you to do all this. Check the manual